app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -16,7 +16,7 @@ def as_json
           status: status,
           phase: application_form.phase,
           updated_at: application_choice.updated_at.iso8601,
-          submitted_at: application_form.submitted_at.iso8601,
+          submitted_at: application_form.submitted_at&.iso8601,
```

Comment:
Perhaps we need a guard within `SingleApplicationPresenter` to return `nil` if the application choice is not in a state visible to providers, and therefore not worry about missing values. What do you think?

Reply:
I like the idea. I don't think the presenter should `nil`, though — might try skipping the cache?